internal/logging/logging.go

```diff
@@ -23,7 +24,8 @@ const (
 	// log level to use when debug is true
 	verboseLevel = log.DebugLevel
 	// time format the out logging uses
-	timeFmt = "2006/01/02 15:04:05.000000"
+	timeFmt               = "2006/01/02 15:04:05.000000"
+	invalidFileDescriptor = ^(uintptr(0))
```

Comment:
nit: feels like this should be exported so it can be referenced outside of the package instead of having to know what the value is.

Reply:
Resolved in 0443e9e